### PR TITLE
MRG: Add "verbose" parameter to pick_channels() method

### PR DESCRIPTION
**Pull request by hoechenberger**
This allows to e.g. to suppress messages about removed projectors.

### CODE REVIEW

File: mne/channels/channels.py [hunk 1]
```diff
@@ -692,7 +692,8 @@ def pick_types(self, meg=False, eeg=False, stim=False, eog=False,
 
         return self
 
-    def pick_channels(self, ch_names, ordered=False):
+    @verbose
+    def pick_channels(self, ch_names, ordered=False, verbose=None):
```
Comment (larsoner) [inline mne/channels/channels.py:696]:
```suggestion
    def pick_channels(self, ch_names, ordered=False, *, verbose=None):
```

Comment (hoechenberger) [reply mne/channels/channels.py:696]: Whoops, good nit**pick**

**Comment by hoechenberger** [comment]
This should be good to merge now

**Comment by larsoner** [comment]
Thanks @hoechenberger !

**Pull request merged**